src/lib/seam/connect/client.ts

```diff
@@ -44,20 +70,67 @@ export class SeamHttp {
   ): SeamHttp {
     const opts = { ...options, clientSessionToken }
     if (!isSeamHttpOptionsWithClientSessionToken(opts)) {
-      throw new InvalidSeamHttpOptionsError('Missing clientSessionToken')
+      throw new SeamHttpInvalidOptionsError('Missing clientSessionToken')
     }
     return new SeamHttp(opts)
   }
 
-  // TODO
-  // static fromPublishableKey and deprecate getClientSessionToken
+  get accessCodes(): SeamHttpAccessCodes {
```

Comment:
Adding these getters are the only manual step needed when a new class is generated. In principal this code block could be generated and injected as well. I suppose the entire file could be generated as well.